packages/bsky/tests/auto-moderator/fuzzy-matcher.test.ts

```diff
@@ -1,165 +0,0 @@
-import { TestNetwork, SeedClient } from '@atproto/dev-env'
-import { FuzzyMatcher, encode } from '../../src/auto-moderator/fuzzy-matcher'
-import basicSeed from '../seeds/basic'
-import { AtpAgent } from '@atproto/api'
-import { ImageInvalidator } from '../../src/image/invalidator'
-
-describe('fuzzy matcher', () => {
```

Comment:
hmmm ripping out automod stuff is definitely going to clash with my work, but hopefully by the time we're landing this branch, the separate automod service is in a good enough spot that it's alright 👌

Reply:
In my view we're ripping everything out in this PR to match the end state we want for the appview frontends.  It's hard to imagine how automod functionality would remain here in v2, so I think we nix it and just handle the merge conflict in this branch at the next opportunity, even if it might be a little while.